spec/requests/registrations_spec.rb

```diff
@@ -0,0 +1,18 @@
+RSpec.describe 'Registrations', type: :request do
```

Comment:
I would prefer this match the formatting of the other request specs.

Reply:
Can you be specific? I modeled this after `spec/requests/organizations_spec.rb`

Reply:
Ah, it wasn't clear to me what the test was focused on before, but this looks good.